math/perfectnumber_test.go

```diff
@@ -0,0 +1,144 @@
+package math
+
+import (
+	"testing"
+)
+
+//example data verified with https://oeis.org/A001065/b001065.txt
+var sumOfProperDivisorsExamples = []struct {
```

Comment:
This is a mutable value, try to change that into a return of a function so that this variable is always deterministic in multiple tests.